### ISSUE TITLE
Wasserstein Distance add to loss function

### ISSUE THREAD

**Issue by muellerdo** [frankkramer-lab/aucmedi#204]
Try out combination of cross-entropy/focal, dice/tversky, distance measure like wasserstein.

Run extensive experiments on various datasets. Compare loss performances.

Interesting for AutoML
